gtests/test_nopython_serializable.cpp

```diff
@@ -322,4 +322,6 @@ TEST(Json, deserialize_with_unordered_map)
     EXPECT_EQ(item.pet_map["cat"].is_cat, true);
 }
 
-} // namespace modmesh
\ No newline at end of file
+} // namespace modmesh
+
+// vim: set ff=unix fenc=utf8 et sw=4 ts=4 sts=4:
```

Comment:
missing newline.

Reply:
It seems that you miss to add a newline after the vim modeline.

<img width="506" alt="image" src="https://github.com/user-attachments/assets/3c3c6666-a124-4677-b5c5-724c569db83b">